tests/engine/test_arg_utils.py

```diff
@@ -80,12 +96,16 @@ class DummyConfigClass:
     """Optional bool with default None"""
     optional_literal: Optional[Literal["x", "y"]] = None
     """Optional literal with default None"""
-    tuple_n: tuple[int, ...] = field(default_factory=lambda: (1, 2, 3))
-    """Tuple with default (1, 2, 3)"""
-    tuple_2: tuple[int, int] = field(default_factory=lambda: (1, 2))
-    """Tuple with default (1, 2)"""
-    list_n: list[int] = field(default_factory=lambda: [1, 2, 3])
-    """List with default [1, 2, 3]"""
+    tuple_n: tuple[int, ...] = field(default_factory=tuple)
+    """Tuple with variable length"""
+    tuple_2: tuple[int, int] = field(default_factory=tuple)
```

Comment:
It looks like there's some mypy faillures here? (at least according to the diff viewer)

Reply:
Just fixed locally